Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
I think we have the same interpretation... the original version only reformats the 'current handler' (I think the one containing the beginning of the selection). Your change makes it do the correct thing by reformatting everything... or am I wrong?